index.md

```diff
@@ -6,4 +6,4 @@ layout: page
 
 ## Shutter website
 
-Something meaningful should be written here.
\ No newline at end of file
+Shutter is a feature-rich screenshot program for Linux based operating systems such as Ubuntu. You can take a screenshot of a specific area, window, your whole screen, or even of a website – apply different effects to it, draw on it to highlight points, and then upload to an image hosting site, all within one window. Shutter is free, [open-source](http://www.opensource.org/docs/definition.php), and licensed under [GPL v3](http://www.gnu.org/licenses/gpl-3.0.html).
```

Comment:
Is anything stopping it from working on non-linux? like freebsd

Reply:
Not sure, but the focus is on Linux primarily. Happy if others can run it too.